pcsx2-qt/Settings/GraphicsSettingsWidget.cpp

```diff
@@ -732,16 +733,20 @@ GraphicsSettingsWidget::GraphicsSettingsWidget(SettingsWindow* dialog, QWidget*
 	// Recording tab
 	{
 		dialog->registerWidgetHelp(m_ui.videoCaptureCodec, tr("Video Codec"), tr("Default"), tr("Selects which Video Codec to be used for Video Capture. "
-		
-		"<b>If unsure, leave it on default.<b>"));
+
+																								"<b>If unsure, leave it on default.<b>"));
+
+		dialog->registerWidgetHelp(m_ui.videoCaptureFormat, tr("Video Format"), tr("Default"), tr("Selects which Video Format to be used for Video Capture. If by chance the codec does not support the format, the first format available will be used. "
+
+																								  "<b>If unsure, leave it on default.<b>"));
 
 		dialog->registerWidgetHelp(m_ui.videoCaptureBitrate, tr("Video Bitrate"), tr("6000 kbps"), tr("Sets the video bitrate to be used. "
-		
-		"Larger bitrate generally yields better video quality at the cost of larger resulting file size."));
+
+																									  "Larger bitrate generally yields better video quality at the cost of larger resulting file size."));
 
 		dialog->registerWidgetHelp(m_ui.videoCaptureResolutionAuto, tr("Automatic Resolution"), tr("Unchecked"), tr("When checked, the video capture resolution will follows the internal resolution of the running game.<br><br>"
-		
-		"<b>Be careful when using this setting especially when you are upscaling, as higher internal resolution (above 4x) can results in very large video capture and can cause system overload.</b>"));
+
+																													"<b>Be careful when using this setting especially when you are upscaling, as higher internal resolution (above 4x) can results in very large video capture and can cause system overload.</b>"));
```

Comment:
If you're going to let clang-format align things, could you make sure they don't start 2/3 of the way across the screen?

(Also please format in a separate commit)